packages/mdc-ripple/types.ts

```diff
@@ -21,15 +21,15 @@
  * THE SOFTWARE.
  */
 
-export interface Point {
+export interface MDCRipplePoint {
```

Comment:
Can you explain why you wanted to leave RippleCapableSurface alone "for backcompat", given that it's a type and meanwhile there are other types that no longer exist (some of which I've pointed out)?

Reply:
I assumed it was more likely to be used by Closure clients, but it looks like I was wrong. Renamed to `MDCRippleCapableSurface`.